portal/src/main/resources/application.conf

```diff
@@ -35,4 +35,10 @@ database {
   liquibase {
     changeLogResourcePath = "liquibase/changelog/changelog-master.xml"
   }
+
+  gitlab {
+    url = "https://gitlab.com/api/v4/"
+    path = "AdminLogin%2F" //TODO: Change AdminLogin for real GitLab login
+    token = "XXXX" //TODO: Change for real GitLab private token
```

Comment:
TODO doesn't look like a good idea :) It should be environment variables

Reply:
Probably yes

Reply:
Fixed in the last commit.